qualtop/wizard/account_invoice_refund.py

```diff
@@ -0,0 +1,26 @@
+from odoo import models, api
+
+
+class AccountInvoiceRefund(models.TransientModel):
+    _inherit = "account.invoice.refund"
+
+    @api.multi
+    def invoice_refund(self):
+        res = super(AccountInvoiceRefund, self).invoice_refund()
+        data_refund = self.read(['filter_refund'])[0]['filter_refund']
+        if data_refund == 'refund' and res.get('domain', False):
```

Comment:
The original method returns boolean for a few cases:
 - https://github.com/odoo/odoo/blob/ced9156a97ea502050b4e827ef2237fa2cd9410f/addons/account/wizard/account_invoice_refund.py#L120

You should check it

Reply:
The unique case where this modification is executed is when data_refund is 'refund', this is:

<img width="1234" alt="skitch-6" src="https://user-images.githubusercontent.com/11341058/43802089-d2928e52-9a62-11e8-9cd6-a13564b9f04c.png">

In this case returns [here](https://github.com/odoo/odoo/blob/ced9156a97ea502050b4e827ef2237fa2cd9410f/addons/account/wizard/account_invoice_refund.py#L119)